fdbclient/NativeAPI.actor.cpp

```diff
@@ -2989,6 +2989,12 @@ void Transaction::setOption( FDBTransactionOptions::Option option, Optional<Stri
 				TraceEvent(SevWarn, "DebugTransactionIdentifierNotSet").detail("Error", "Debug Transaction Identifier option must be set before logging the transaction");
 				throw client_invalid_operation();
 			}
+			if (info.debugID.present()) {
+				TraceEvent(SevInfo, "TransactionBeingTraced")
+					.detail("DebugTransactionID", trLogInfo->identifier)
+					.detail("ServerTraceID", info.debugID.get().toString());
+
+			}
```

Comment:
Oh, I see now. I guess if logging them is dependent on having set a debug transaction ID, then I guess this makes the most sense one block further up in `DEBUG_TRANSACTION_IDENTIFIER`.

Reply:
Correct. That's where I meant to put it and just lost all context in between. It should be fixed now.